ios/Podfile

```diff
@@ -17,6 +17,13 @@ end
 target 'JitsiMeet' do
   project 'app/app.xcodeproj'
 
+  # React Native and its dependencies
+  #
+
+  use_react_native!(
```

Comment:
The app doesn't need this. The only part that uses Hermes, as part of RN is the SDK.